### ISSUE TITLE
Unable to go to zen-browser.app when using zen-browser, i can go to the website using other browsers

### ISSUE THREAD

**Issue by tijnsemmekrot** [zen-browser/desktop#3345]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

When i navigate to the website zen-browser.app i get this message
I don't know why i get this, but this is problably a bug. This is also the case for firefox
<img width="1549" alt="image" src="https://github.com/user-attachments/assets/4e8d4d08-9353-497d-8b33-3f78432b8090">

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

zen twighlight and current

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by tijnsemmekrot** [comment]
I am not sure if there changed anything. However it works for me right now. I didn't change anything. However I did have this error for over a week so maybe some others have this issue

**Comment by hg0428** [comment]
Yes, I have this issue. But for me this happens on all browsers.
